website/client/components/static/faq.vue

```diff
@@ -4,7 +4,8 @@
       .col-12.col-md-6.offset-md-3
         h1 {{ $t('frequentlyAskedQuestions') }}
         .faq-question(v-for='(heading, index) in headings')
-          h2.accordion(@click='setActivePage(heading)') {{ $t(`faqQuestion${index}`) }}
+          h2.accordion(:ref='uniqueRef(index)')
+            a(v-on:click.stop.prevent="" href="#", :id='heading', @click='setActivePage(heading)') {{ $t(`faqQuestion${index}`) }}
```

Comment:
you could just use `cursor: pointer;` in your style, instead of adding `a` just for the cursor